docs/source/installation-and-deployment/downgrading.rst

```diff
@@ -0,0 +1,63 @@
+==============================
+Downgrading an OpenKAT install
+==============================
+
+Downgrading is not officially supported, however if you do need to downgrade these tips might help.
+Please also be advised that running mixed versions of the OpenKAT parts can lead to unspecified behaviour.
+
+Make Backups of your `postgresql` database before attempting rollbacks if you value your data.
+
+Downgrading the scheduler / Mula:
+=================================
+
+If you have deployed a previous version of the Scheduler, and are presented with Alembic migration errors about migration files being missing, this indicates that the version you initially installed (be it Main, or a specific release) has ran migrations that are now no longer available for the container or for the process.
+Example:
+
+.. code-block:: sh
+
+  # docker log -f nl-kat-coordination-scheduler-1
+  INFO  [alembic.runtime.migration] Context impl PostgresqlImpl.
+  INFO  [alembic.runtime.migration] Will assume transactional DDL.
+  ERROR [alembic.util.messaging] Can't locate revision identified by '0010'
+  FAILED: Can't locate revision identified by '0010'
+
+
+In this example migration 10 used to be available, and has been applied, but now can no longer be found.
+
+To fix this:
+
+1. Start a scheduler container from the original (newer) version which contains the migration revision id `0010` (for our example).
+
+   If you ran a specific branch, checkout that branch, or some other similar branch that contains the same revision file.
+   `alembic` needs this file to know how to migrate back to a lower version.
+
+2. Obtain an interactive shell in the scheduler container. The `docker` command would be:
+
+.. code-block:: sh
+
+  docker exec -it <container_id> /bin/bash
+  docker exec -it nl-kat-coordination-scheduler-1 /bin/bash
```

Comment:
This won't work if you have a different directory name. For this reason and among others and as recommended for docker compose projects we've replaced all docker exec and run commands examples and references to use docker compose instead.

```suggestion
  docker compose exec <service> /bin/bash
  docker compose exec scheduler /bin/bash
```